src/main/java/com/scalar/cassy/scheduler/CassyClient.java

```diff
@@ -0,0 +1,144 @@
+package com.scalar.cassy.scheduler;
+
+import com.scalar.cassy.config.BackupType;
+import com.scalar.cassy.exception.BackupException;
+import com.scalar.cassy.rpc.BackupListingRequest;
+import com.scalar.cassy.rpc.BackupListingResponse;
+import com.scalar.cassy.rpc.BackupRequest;
+import com.scalar.cassy.rpc.BackupResponse;
+import com.scalar.cassy.rpc.CassyGrpc;
+import com.scalar.cassy.rpc.ClusterListingRequest;
+import com.scalar.cassy.rpc.ClusterListingResponse;
+import com.scalar.cassy.rpc.OperationStatus;
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+public class CassyClient {
+  private final CassyGrpc.CassyBlockingStub blockingStub;
+
+  public CassyClient() {
+    this(
+        ManagedChannelBuilder.forAddress(
+                System.getenv().getOrDefault("CASSY_SCHEDULER_HOST", "localhost"),
+                Integer.parseInt(System.getenv().getOrDefault("CASSY_SCHEDULER_PORT", "20051")))
+            .usePlaintext());
+  }
+
+  public CassyClient(ManagedChannelBuilder<?> channelBuilder) {
+    Channel channel = channelBuilder.build();
+    blockingStub = CassyGrpc.newBlockingStub(channel);
+  }
+
+  public void takeClusterSnapshot(String clusterId, int timeout) throws Exception {
```

Comment:
It doesn't seek like reporting the result back to a client.
I think it should report like below otherwise users have to check the state database every time to see if a command properly worked, which defeats the purpose of the tool.
```
IP x: SUCCEEDED
IP y: SUCCEEDED
IP y: FAILED
```

And the exit code should be properly set to follow the common practice.
If all the specified nodes' backups are properly taken, it should be `0` and otherwise `1`.

This comment applies to all the other commands.